vlasiator.cpp

```diff
@@ -1110,6 +1113,9 @@ int main(int argn,char* args[]) {
          // Make sure the ionosphere communicator is up-to-date, in case inner boundary cells
          // moved.
          SBC::ionosphereGrid.updateIonosphereCommunicator(mpiGrid, technicalGrid);
+
```

Comment:
Let's also instead of here, call it inside `balanceLoad()` inside `grid.cpp`, at line 607 or so (at the same place as re-calculating local cells cache)

Reply:
( I do appreciate that it's in the same place as the updateIonosphereCommunicator, but still, that would be my recommendation)